src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -1127,6 +1130,7 @@ void DeformableConvolution::DefConvRefExecutor::exec(const float* src, const flo
     const size_t group_wei_stride = weiStrides[0] * OC;
     auto compKer = [=](int g, int mb, int oc, int oh, int ow) {
         float d = 0;
+        float w11 = 0, w12 = 0, w21 = 0, w22 = 0;
```

Comment:
why these variables are declared separatelly with the definition? Seems like they are not reused antwhere.

Reply:
Kind of minor optimization to prevent repeated initialization on every step of the cycle. Anyway so as on modern compilators it must be resolved correctly, I moved it inside the cycle to make code clearly.